examples/bri-3/src/bri/identity/bpiSubjectAccounts/agents/bpiSubjectAccountsStorage.agent.ts

```diff
@@ -19,24 +25,18 @@ export class BpiSubjectAccountStorageAgent extends PrismaService {
       throw new NotFoundException(NOT_FOUND_ERR_MESSAGE);
     }
 
-    const creatorBpiSubject = new BpiSubject(
-      bpiSubjectAccountModel.CreatorBpiSubject.id,
-      bpiSubjectAccountModel.CreatorBpiSubject.name,
-      bpiSubjectAccountModel.CreatorBpiSubject.description,
-      bpiSubjectAccountModel.CreatorBpiSubject.type,
-      bpiSubjectAccountModel.CreatorBpiSubject.publicKey,
-    );
-    const ownerBpiSubject = new BpiSubject(
-      bpiSubjectAccountModel.OwnerBpiSubject.id,
-      bpiSubjectAccountModel.OwnerBpiSubject.name,
-      bpiSubjectAccountModel.OwnerBpiSubject.description,
-      bpiSubjectAccountModel.OwnerBpiSubject.type,
-      bpiSubjectAccountModel.OwnerBpiSubject.publicKey,
-    );
-    return new BpiSubjectAccount( // TODO: Write generic mapper prismaModel -> domainObject
+    return new BpiSubjectAccount(
```

Comment:
we cant invoke mapper on BpiSubjectAccount and set the corresponding BpiSubject mapping in the profile or via automap?

Reply:
This can't be done implicitely as we are mapping from prisma model to domain model. As discussed, once changes from @skosito are done, this can be replaced with the automap call.